### PR TITLE
Use cf7 commands for the cf CLI commands

### PR DESCRIPTION
**Pull request by idavidmcdonald**
As we want to upgrade from cf6 to cf7 CLI

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -180,22 +181,31 @@ cf-deploy: ## Deploys the app to Cloud Foundry
...
-	cf push -f <(CF_APP=notify-admin-prototype-2 make -s generate-manifest)
+	make -s CF_APP=notify-admin-prototype-2 generate-manifest > ${CF_MANIFEST_PATH}
+	cf push notify-admin-prototype-2 --strategy=rolling -f ${CF_MANIFEST_PATH}
+	rm ${CF_MANIFEST_PATH}
 
 .PHONY: cf-rollback
 cf-rollback: cf-target ## Rollbacks the app to the previous release
-	cf v3-cancel-zdt-push ${CF_APP}
+	cf cancel-deployment ${CF_APP}
+	rm ${CF_MANIFEST_PATH}
```
Comment (leohemsted) [inline Makefile:208]: should have realised in the API pull request, my bad. i think this rm command should have the `-f` arg. That way, if the file doesn't exist, there's no output and the makefile doesn't fail. It's most relevant to this rollback command, but can't hurt in the other commands I think. If you agree I can update the api makefile

```suggestion
	rm -f ${CF_MANIFEST_PATH}
```

**Review by leohemsted — COMMENTED** [review]
One minor comment

File: Makefile (hunk 1 above)
Comment (idavidmcdonald) [reply Makefile:208]: Agreed

Comment (idavidmcdonald) [reply Makefile]: I've squashed and pushed up changes

**Review by leohemsted — APPROVED** [review]

**Pull request merged**